### PR TITLE
Fixed artifacts handling + added tests logs aggregation

### PR DESCRIPTION
**Pull request by rafie**

### CODE REVIEW

File: test/includes.py [hunk 1]
```diff
@@ -24,8 +24,8 @@
 COV = os.environ.get("COV") != "0" and os.environ.get("COV") != "0"
 DEVICE = os.environ.get('DEVICE', 'CPU').upper().encode('utf-8', 'ignore').decode('utf-8')
 VALGRIND = os.environ.get("VALGRIND") == "1"
-print(f"Running tests on {DEVICE}\n")
-print(f"Using a max of {MAX_ITERATIONS} iterations per test\n")
+print("Running tests on {}\n".format(DEVICE))
```
Comment (rafie) [inline test/includes.py:27]: Some poor platforms are still below the Python 3.6 threshold. :-(

**Comment by rafie** [comment]
Tested with RS/nightly build 2066, works Ok.

**Review by lantiga — APPROVED** [review]
LGTM (although I'm partially unqualified :-) )

**Review by filipecosta90 — APPROVED** [review]

**Comment by filipecosta90** [comment]
> LGTM (although I'm partially unqualified :-) )

Same her. Thank you for pushing this big change Rafi :) 
Lets cherry-pick to 1.0 also.

**Pull request merged**